api/app/signals/apps/dataset/sources/sia_stadsdeel.py

```diff
@@ -61,6 +61,36 @@ def _load_amsterdamse_bos_geometry(self):
         geometries[0].srid = 4326  # Set SRID to WGS84
         return geometries[0]
 
+    def _combine_ouder_amstel_and_oost(self):
+        """
+        Meldingen in "Ouder-Amstel" should be tagged as stadsdeel "Oost". Therefore we need to combine the geometry of
+        Ouder-Amstel and Oost. In the Jira of Amsterdam see ticket SIG-4436
+        """
+        try:
+            sia_stadsdeel_oost = Area.objects.get(name__iexact='oost', _type__name=self.area_type)
+        except Area.DoesNotExist:
+            return
+
+        try:
+            cbs_gemeente_ouder_amstel = Area.objects.get(name__iexact='Ouder-Amstel',
+                                                         _type__code__in=['cbs-gemeente-2019', 'cbs-gemeente-2021'])
+        except Area.DoesNotExist:
+            return
+
+        # Combine SIA stadsdeel Oost and CBS Gemeente Ouder Amstel
+        combined_geometry = sia_stadsdeel_oost.geometry + cbs_gemeente_ouder_amstel.geometry
+
+        # Only handle non-pathological cases. We want the difference between Zuid
```

Comment:
Nitpick, rephrase (still references stadsdeel "Zuid" which is handled in another method below).